chunkset_tpl.h

```diff
@@ -176,7 +176,18 @@ Z_INTERNAL uint8_t* CHUNKMEMSET(uint8_t *out, unsigned dist, unsigned len) {
 }
 
 Z_INTERNAL uint8_t* CHUNKMEMSET_SAFE(uint8_t *out, unsigned dist, unsigned len, unsigned left) {
+#ifdef UNALIGNED64_OK
+    static const uint32_t align_mask = 3;
+#else
+    static const uint32_t align_mask = 7;
+#endif
     len = MIN(len, left);
+    uint8_t *from = out - dist;
+    while (((uintptr_t)out & align_mask) && (len > 0)) {
+        *out++ = *from++;
+        --len;
+        --left;
+    }
     if (left < (unsigned)(3 * sizeof(chunk_t))) {
         uint8_t *from = out - dist;
```

Comment:
Redeclaration of variable with same name.

Reply:
Thanks. Forgot to remove it, since I am working with different repo.